src/tally/KDEKernel.cpp

```diff
@@ -49,9 +51,11 @@ double KDEKernel::boundary_correction(const double* u,
 
   // solve for the boundary correction factor
   if (num_corrections == 1) {
+
```

Comment:
are those blank lines required by style ?

Reply:
Not sure, but `astyle` doesn’t complain, so I hold my breath and cross my fingers....

Reply:
we really need to move away from `astyle` :)